BTT/BeyondTheTutor/BeyondTheTutor/Models/Student.cs

```diff
@@ -19,10 +18,10 @@ public partial class Student
         [StringLength(50)]
         public string LastName { get; set; }
 
-        public DateTime GraduatingYear { get; set; }
+        public short GraduatingYear { get; set; }
```

Comment:
idk why it says short here, but it should be INT16 but i think it might be a code thing..